File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/messagemgr/EventMessage.java
package com.tencentcs.iotvideo.messagemgr;

/* loaded from: classes2.dex */
public class EventMessage extends Message {
    public String data;
    public String topic;

    public EventMessage(int type, String topic, String data) {
        super(type, 0L, 0);
        this.topic = topic;
        this.data = data;
    }

    @Override
    public String toString() {
        StringBuilder sb2 = new StringBuilder("EventMessage{topic='");
        sb2.append(this.topic);
        sb2.append("', data='");
        sb2.append(this.data);
        sb2.append("'}");
        return sb2.toString();
    }
}

File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/messagemgr/EventMessage.kt
package com.tencentcs.iotvideo.messagemgr

class EventMessage(type: MessageType, var topic: String, var data: String) : Message(type, 0L, 0) {
    override fun toString(): String {
        val sb2 = StringBuilder("EventMessage{topic='")
        sb2.append(this.topic)
        sb2.append("', data='")
        sb2.append(this.data)
        sb2.append("'}")
        return sb2.toString()
    }
}
